README.md

```diff
@@ -14,10 +14,3 @@ To use:
 3. In iTunes Connect, create and configure any IAP products for test.
 4. Enumerate the corresponding IAP identifiers in the ProductIdentifiers.plist.
 5. Build and run.
-
-
-Licence
```

Comment:
Would it be clearer to make a mention of the MIT licence here in the readme too?